### PR TITLE
Remove filter because we are already filtering

### PR DESCRIPTION
**Pull request by dshanske**
#399 This removes the filtering before adding the webmention, on the basis we do our own filtering in webmention_sanitize_html and restores it afterward.

### CODE REVIEW

**Review by pfefferle — COMMENTED** [review]
Shouldn't we instead remove our kses? Isn't that easier?

File: includes/class-receiver.php [hunk 1]
```diff
@@ -351,9 +351,19 @@ public static function post( $request ) {
...
 
-			// save comment
+			// save comment but remove content filtering because we filter our own content
+			remove_filter( 'pre_comment_content', 'wp_filter_post_kses' );
+			remove_filter( 'pre_comment_content', 'wp_filter_kses' );
+
 			$commentdata['comment_ID'] = wp_new_comment( $commentdata, true );
 
+			// restore filter after add
+			if ( current_user_can( 'unfiltered_html' ) ) {
```
Comment (pfefferle) [inline includes/class-receiver.php:361]: This is an API request, so there is no current_user!?!

**Comment by dshanske** [comment]
> Shouldn't we instead remove our kses? Isn't that easier?

#399 suggests that the built-in WordPress filtering is taking out elements we would keep in.

File: includes/class-receiver.php (hunk 1 above)
Comment (pfefferle) [reply includes/class-receiver.php:361]: what if there is no current_user? will it fail gracefully?

Comment (dshanske) [reply includes/class-receiver.php:361]: Yes.

**Review by pfefferle — APPROVED** [review]
LGTM, thanks @dshanske

**Pull request merged**